### PR TITLE
Created stop watch

### PR DESCRIPTION
**Pull request by illiashcherbak**
- [DEMO LINK](https://illiashcherbak.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/utils/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$watch-size: 600px;
+$center-size: 30px;
```
Comment (APetrinko) [inline src/styles/utils/_variables.scss:2]: cl p10 
u use this variable only once

File: src/styles/blocks/stop-watch.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  &__second-hand {
+    width: $hand-width;
+    height: 45%;
+
+    @include hand-position;
+
+    border-radius: $hand-radius;
+    background-color: #6495ed;
+
+    animation: go-around 60s linear infinite;
```
Comment (APetrinko) [inline src/styles/blocks/stop-watch.scss:42]: As for me, u can create mixin for this value

@mixin name ($duration) {
 animation: go-around $duration linear infinite;
}

Its up to u)

**Review by APetrinko — APPROVED** [review]
Gj 👍 
only 1 little issue

File: src/styles/blocks/stop-watch.scss (hunk 2 above)
Comment (ivan-kadykalo) [reply src/styles/blocks/stop-watch.scss:42]: Yea, it is really good advice. DRY

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Good job, but need to fix the rest of the comments

File: src/styles/blocks/stop-watch.scss (hunk 2 above)
Comment (illiashcherbak) [reply src/styles/blocks/stop-watch.scss:42]: Thank you so much for your advice and for checking my work! I'll fix it now

File: src/styles/blocks/stop-watch.scss [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    border-radius: $hand-radius;
+    background-color: #6495ed;
+
+    @include animation(60s);
+  }
+
+  @keyframes go-around {
+    0% {
+      transform: rotate(0);
+      transform-origin: bottom center;
```
Comment (ivan-kadykalo) [inline src/styles/blocks/stop-watch.scss:48]: This property doesn't change during animation, so it is redundant or should be in another place

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]

File: src/styles/blocks/stop-watch.scss (hunk 3 above)
Comment (illiashcherbak) [reply src/styles/blocks/stop-watch.scss:48]: When I delete 0% with its properties, it breaks. Or I didn't catch what you mean

Comment (ivan-kadykalo) [reply src/styles/blocks/stop-watch.scss:48]: I mean transform-origin. It just repeated for 0% and 100%

**Review by ivan-kadykalo — APPROVED** [review]
🔥

File: src/styles/blocks/stop-watch.scss (hunk 3 above)
Comment (illiashcherbak) [reply src/styles/blocks/stop-watch.scss]: Thank you Ivan

**Pull request closed without merging**